src/pages/Events/index.jsx

```diff
@@ -62,273 +50,71 @@ const Events = () => {
+        <section className=" bg-event-hero-bg bg-event-bg">
+          <div className="w-full min-h-[600px] flex flex-col justify-center text-white event-hero">
+            <h1 className="hero-heading capitalize font-bold text-[48px] md:text-[56px] 2md:text-[64px] text-center leading-[150%] mx-auto">
+              events
+            </h1>
+
+            <p className="m-0 mt-4 text-center max-w-[720px] text-lg md:text-2xl mx-auto leading-[30px]">
+              She Code Africa is a registered non-profit organization dedicated
+              to empowering and celebrating girls and women in technology
+              across  Africa
+            </p>
           </div>
         </section>
 
-        <section className="mt-24 w-full text-secondary-dark-brown">
-          <h2 className="text-3xl font-bold mb-0 lg:my-18 lg:text-5xl text-center">
-            Up coming events
+        <section className="mt-24 w-full text-primary-dark-brown">
+          <h2 className="text-3xl font-semibold mb-0 lg:my-18 lg:text-5xl text-center">
+            Upcoming
           </h2>
-          <div className=" w-[90%] mx-auto md:w-[80%] mt-16">
-            {upcomingEvents.map((event, index) => {
-              return (
-                <div
-                  className={`w-full flex flex-col ${
-                    (index + 1) % 2 !== 0
-                      ? "2md:flex-row-reverse"
-                      : "2md:flex-row"
-                  } gap-14 2md:items-center 2md:justify-between mb-10`}
-                  key={event.id}
-                >
-                  <article className="w-full md:max-w-[480px] mx-auto 2md:mx-0">
-                    <h3 className="text-2xl md:text-3xl 2md:text-[40px] capitalize font-semibold text-center md:text-justify">
-                      {event.heading}
-                    </h3>
-
-                    <p className="mt-7 text-base md:text-xl text-center md:text-justify">
-                      {event.textContent}
-                    </p>
 
-                    <div className="flex mt-[28px] justify-center md:justify-start">
-                      <Link
-                        to="#"
-                        className="bg-primary-main-pink text-white py-[14px] px-8 rounded-lg capitalize text-sm transition duration-300 hover:bg-transparent hover:text-primary-main-pink border-2 border-primary-main-pink"
-                      >
-                        {event.pathname}
-                      </Link>
-                    </div>
-                  </article>
-                  <figure className="m-0 p-0 w-full h-[482px] max-w-[482px] 2md:max-w-[482px] mx-auto 2md:mx-0">
-                    <img
-                      src={event.eventImage}
-                      alt="event-pic"
-                      className="w-full h-full object-cover"
-                    />
-                  </figure>
-                </div>
-              );
-            })}
-          </div>
+          {!isLoading && isSuccess && isFetched && (
```

Comment:
check for `isFetched && isSuccess` remove` !isLoading` when `isFetched` then `isLoading` becomes false so this is redundant